static/app/views/performance/data.tsx

```diff
@@ -441,6 +530,77 @@ function generateBackendPerformanceEventView(
   return eventView;
 }
 
+function generateMobilePerformanceEventView(
```

Comment:
Not really a comment for this PR, just an aside; now that we have a quite a few of these event view generation functions we should probably DRY 🤔